code/modules/clothing/suits/armor.dm

```diff
@@ -338,6 +346,26 @@
 	acid = 100
 	wound = 15
 
+// Marauder Corps / SyndieParatrooper Suit
+/obj/item/clothing/suit/armor/swat/gorlex
+	name = "tactical armour suit"
+	desc = "A tactical full-body armour suit, dubbed the \"NOVA\". Spaceproof, with a decent armour and emblazed with black and red colours. Property of Gorlex Security Consulting, LLC."
+	icon_state = "marauder_suit"
+	armor_type = /datum/armor/armor_nova
+
+/datum/armor/armor_nova
+	melee = 35
+	bullet = 30
+	laser = 30
+	energy = 40
+	bomb = 25
+	bio = 90
+	fire = 100
+	acid = 100
+	wound = 10
+
+//All of the armor below is mostly unused
```

Comment:
This comment really doesn't help much at all, and is somewhat misleading at that. While there are definitely some unused bits of content in here that probably needs to be cleaned up, there is definitely some existing objects that are being used in the game presently, like durathread vests. It wasn't something you introduced, but you can probably get rid of it since it isn't even true.